### PR TITLE
AI Test Toolkit: Improve command line tool

### PR DESCRIPTION
**Pull request by t-prda**
#### Summary 
1. Make the command line card page accessible from tell me
1. Show the test method lines on the page for the selected suite
1. Support basic filtering using test method line no. for running the next test.
1. Expose the suite and test method line description in the AI Log Entry API
1. Create a new API to access the test method lines

Screenshot:
image

#### Work Item(s) 
Fixes [AB#558560](https://dynamicssmb2.visualstudio.com/1fcb79e7-ab07-432a-a3c6-6cf5a88ba4a5/_workitems/edit/558560)

### CODE REVIEW

File: src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al [hunk 1]
```diff
@@ -222,39 +283,62 @@ page 149042 "AIT CommandLine Card"
...
+        TestSuiteCodeRequiredErr: Label 'Test Suite Code is required to run the suite.';
+    begin
+        if AITCode = '' then
+            Error(TestSuiteCodeRequiredErr);
     end;
 
     local procedure RefreshNoOfPendingTests(): Integer
     var
-        AITTestMethodLine: Record "AIT Test Method Line";
+        Rec2: Record "AIT Test Method Line";
```
Comment (stkillen) [inline src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al:320]: Minor: why this name change?

File: src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al [hunk 2]
```diff
@@ -222,39 +283,62 @@ page 149042 "AIT CommandLine Card"
...
+        if AITTestMethodLine.FindFirst() then begin
             AITTestSuiteMgt.RunAITestLine(AITTestMethodLine, false);
+            UpdateAITestMethodLines();
+        end;
+    end;
 
-        RefreshNoOfPendingTests();
+    local procedure VerifyTestSuiteCode()
+    var
+        TestSuiteCodeRequiredErr: Label 'Test Suite Code is required to run the suite.';
```
Comment (stkillen) [inline src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al:312]: Minor: make this a global variable?

**Review by stkillen — APPROVED** [review]

File: src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al (hunk 1 above)
Comment (t-prda) [reply src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al:320]: Won't fix

File: src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al (hunk 2 above)
Comment (t-prda) [reply src/Tools/AI Test Toolkit/src/AITCommandLineCard.Page.al:312]: Won't fix

**Review by PredragMaricic — APPROVED** [review]

**Pull request merged**